compio-driver/src/sys/iocp/op.rs

```diff
@@ -482,17 +482,53 @@ impl<S: AsFd> OpCode for Connect<S> {
     }
 }
 
+pin_project! {
+    /// Receive data from remote.
+    ///
+    /// It is only used for socket operations. If you want to read from a pipe, use
+    /// [`Read`].
+    pub struct Recv<T: IoBufMut, S> {
+        pub(crate) fd: S,
+        #[pin]
+        pub(crate) buffer: T,
+        pub(crate) flags: i32,
+        pub(crate) slice: WSABUF,
```

Comment:
What about use `SysSlice` here and `SysSlice::null` in `new`? Similar for other ops taking one `WSABUF`.

Reply:
Good idea